fast64_internal/sm64/sm64_geolayout_bone.py

```diff
@@ -197,6 +197,13 @@ def draw(self, context):
             col.box().label(text="This is in blender units.")
             prop_split(col, obj, "culling_radius", "Culling Radius")
 
+        col.prop(obj, "add_shadow")
+        if obj.add_shadow:
+            col.box().label(text="This is not in blender units.")
```

Comment:
it would be more helpful to describe what the unit actually is then than what it isn't :sunglasses: